PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -1017,6 +1017,13 @@ enum Platform { PF_UNKNOWN, PF_X86, PF_X64, PF_IA64, PF_ARM64 };
 	//  	...
 	//  }
 
+	#define NPPM_GETTOOLBARICONSETMODE (NPPMSG + 118)

```

Comment:
Use "NPPM_GETTOOLBARICONSETCHOICE" instead.

Reply:
Implemented in the new commit: "Renamed message to NPPM_GETTOOLBARICONSETCHOICE"